### DISCUSSION TITLE
Repo error after I push to github

### DISCUSSION THREAD

**Discussion by emu-code** [Cyfrin/foundry-full-course-cu#2136] [General]
<img width="633" alt="Screenshot 2024-07-30 at 12 44 26" src="https://github.com/user-attachments/assets/b2dd0796-5796-49b4-90cf-5ab8bf2a26cf">
 

I have just finished the foundry fund me section and I pushed straight to github but I get a red x. I have tried troubleshooting with chatgpt and spent all day on it to no avail. I'd need help with this one my community. Here's the github:

https://github.com/emu-code/foundry-fund-me-pilot

**Comment by EngrPips** [comment]
I think you successfully pushed the code to GitHub because I can see all your code when I visited this `https://github.com/emu-code/foundry-fund-me-pilot`

**Reply by emu-code** [reply]
Yes it is but I'm not enjoying seeing that error, it's not a good sight

**Comment by EngrPips** [comment]
> <img alt="Screenshot 2024-07-30 at 12 44 26" width="633" src="https://private-user-images.githubusercontent.com/58421025/353443695-b2dd0796-5796-49b4-90cf-5ab8bf2a26cf.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjIzNzExMDIsIm5iZiI6MTcyMjM3MDgwMiwicGF0aCI6Ii81ODQyMTAyNS8zNTM0NDM2OTUtYjJkZDA3OTYtNTc5Ni00OWI0LTkwY2YtNWFiOGJmMmEyNmNmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MzAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzMwVDIwMjAwMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTFjNWRmMWVmODUxODBiYTIyNGZiNjUyMDFkMGJiM2Q1ZjcwNGU1NzEwOGQyMWYzYzI5YTU0ZWMzN2VlNzBhNmQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.OHaAw6g4x4lwF2lP6AYue9p6ROe2YrWtJSt-DNRPtfc">
> I have just finished the foundry fund me section and I pushed straight to github but I get a red x. I have tried troubleshooting with chatgpt and spent all day on it to no avail. I'd need help with this one my community. Here's the github:
> 
> https://github.com/emu-code/foundry-fund-me-pilot

Indeed, why not check the details and see why it failed.

**Comment by spencerj411** [comment]
@emu-code Any luck? To be fair, that workflow was generated by foundry; seems to be a template. I don't know if it's even supposed to pass.

**Reply by EngrPips** [reply]
hmm 🤔
